gr-filter/grc/filter_band_pass_filter.block.yml

```diff
@@ -83,6 +84,25 @@ templates:
     - set_taps(firdes.${type.fcn}(${gain}, ${samp_rate}, ${low_cutoff_freq}, ${high_cutoff_freq},
         ${width}, ${win}, ${beta}))
 
+cpp_templates:
+    includes: ['#include <gnuradio/filter/firdes.h>']
+    declarations: 'filter::firdes::sptr ${id};'
+    make: |-
+        this->${id} = filter::${type}::make(
+            ${ interp if str(type).startswith('interp') else decim },
+            firdes.${type.fcn}(
+                ${gain},
+                ${samp_rate},
+                ${low_cutoff_freq},
+                ${high_cutoff_freq},
+                ${width},
+                ${win},
+                ${beta}));
+    link: ['gnuradio-filter']
+    callbacks:
+    - set_taps(firdes::band_reject(${gain}, ${samp_rate}, ${low_cutoff_freq}, ${high_cutoff_freq},
```

Comment:
I don't think `band_reject` is correct here

Reply:
Sorry for the typo, I fixed it